x-pack/platform/plugins/shared/security/server/session_management/session_index.test.ts

```diff
@@ -96,10 +96,77 @@ describe('Session index', () => {
 
       expect(mockElasticsearchClient.indices.deleteTemplate).not.toHaveBeenCalled();
       expect(mockElasticsearchClient.indices.putIndexTemplate).not.toHaveBeenCalled();
-      expect(mockElasticsearchClient.indices.putAlias).not.toHaveBeenCalled();
+      expect(mockElasticsearchClient.indices.putAlias).toHaveBeenCalledWith({
+        index: indexName,
+        name: aliasName,
+      });
       expect(mockElasticsearchClient.indices.create).not.toHaveBeenCalled();
     });
 
+    it('does not create index if alias exists', async () => {
+      mockElasticsearchClient.indices.existsTemplate.mockResponse(false);
+      mockElasticsearchClient.indices.existsIndexTemplate.mockResponse(true);
+      mockElasticsearchClient.indices.exists.mockImplementation(async ({ index }) => {
+        if (index === aliasName) {
+          return true;
+        }
+        return false;
+      });
+
+      await sessionIndex.initialize();
+
+      assertExistenceChecksPerformed();
+
+      expect(mockElasticsearchClient.indices.deleteTemplate).not.toHaveBeenCalled();
+      expect(mockElasticsearchClient.indices.putIndexTemplate).not.toHaveBeenCalled();
+      expect(mockElasticsearchClient.indices.putAlias).toHaveBeenCalledWith({
+        index: indexName,
+        name: aliasName,
+      });
+      expect(mockElasticsearchClient.indices.create).not.toHaveBeenCalled();
+    });
+
+    it('does not create index if index exists', async () => {
+      mockElasticsearchClient.indices.existsTemplate.mockResponse(false);
+      mockElasticsearchClient.indices.existsIndexTemplate.mockResponse(true);
+      mockElasticsearchClient.indices.exists.mockImplementation(async ({ index }) => {
+        if (index === indexName) {
+          return true;
+        }
+        return false;
+      });
+
+      await sessionIndex.initialize();
+
+      assertExistenceChecksPerformed();
```

Comment:
__nit:__ same nit here
```suggestion
      expect(mockElasticsearchClient.indices.exists).toHaveBeenCalledWith({ index: aliasName });
      expect(mockElasticsearchClient.indices.exists).toHaveBeenCalledWith({ index: indexName });
      expect(mockElasticsearchClient.indices.exists).toHaveBeenCalledTimes(2);
```